docs/routing/introduction.md

```diff
@@ -70,6 +70,8 @@ The example above uses multiple links. Each one maps a path (`href`) to a known
 
 Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes is _not_ prefetched.
 
+Also note that when navigating to another page using `<Link />` even tough routing happens client side, the data for that page (via `GetServerSideProps`) _is_ fetched before navigating to the new route.
```

Comment:
Lets just add it to line 71 at the end.

> routes is _not_ prefetched, but it will be once the `<Link />` is clicked.

Reply:
I prefer to be more explicit (your version reads a little bit more complicated to me as a non-native speaker), but that version also works for me.

Reply:
See https://github.com/vercel/next.js/pull/41577#discussion_r999992319